.gitbook/5-oracles/1-operate-oracle-nodes/7-verify-remote-report.md

```diff
@@ -0,0 +1,92 @@
+# Verify Remote Report
+
+# Genesis Oracle
+
+- Status: Draft
+- Created: 2023-01-12
+- Modified: 2023-01-12
+- Authors
+    - Gyuguen Jang <gyuguen.jang@medibloc.org>
+    - Youngjoon Lee <yjlee@medibloc.org>
+    - Hansol Lee <hansol@medibloc.org>
+    - Myongsik Gong <myongsik_gong@medibloc.org>
+    - Inchul Song <icsong@medibloc.org>
+
+
+## Synopsis
+
+This document describes how to validate remote reports generated by Oracle.
+
+## Remote Report
+
+Remote reports are one of the core of our protocol. This remote report must be submitted when the genesis oracle or new oracle participant registers with Panacea. A remote report can verify that the genesis oracle or new oracle participant is the correct oracle. 
```

Comment:
How about some more specific explanation of what a correct oracle is? If you write a few lists with bullets, it seems to better express what a verify report means.

Reply:
Yes. You can put a link to the Confidential Oracle doc if possible.

Reply:
Yes. However, I think we don't have an explan document on how Oracle is valid.
Is there any documentation I'm missing?

Reply:
And I think it would be nice to add that the correct oracle can be determined by a unique ID, and this will change if the oracle is tampered with.

Reply:
Oh, I'm sorry, I found the content in the place @youngjoon-lee talked about.

Reply:
@audtlr24 
I've linked to verifying correct oracle with remote credentials. Would you like to add a description other than this part?

Reply:
Added that if the unique ID is changed, it is not a valid oracle.
https://github.com/medibloc/panacea-core/pull/620/commits/83b89c2064c06e21324d6dfcdc25deb2d083fa52

Reply:
Yeah. Actually, the confidential computing doc doesn't have the exact chapter to be referred to, but the one that you used is good for now.